src/components/views/settings/debug.tsx

```diff
@@ -167,7 +166,7 @@ export const Debug = ({navigation}) => {
   };
 
   return (
-    <Layouts.Basic heading="Debug">
+    <Basic heading="Debug">
```

Comment:
Perhaps not for this PR, but why is this `heading` coded inline instead of using a localized string as the others screens do.